gvr-unittestutils/src/main/java/org/gearvrf/unittestutils/GVRTestableActivity.java

```diff
@@ -31,11 +35,50 @@
     @Override
     protected void onCreate(Bundle bundle) {
         super.onCreate(bundle);
+
+        requestPermissions();
+
         testableMain = new GVRTestableMain();
         setMain(testableMain, "gvr.xml");
         Log.d(TAG, "OnCreate called");
     }
 
+    private void requestPermissions() {
+        if (ContextCompat.checkSelfPermission(this,
+                Manifest.permission.READ_EXTERNAL_STORAGE)
+                != PackageManager.PERMISSION_GRANTED) {
+
+            ActivityCompat.requestPermissions(this,
+                    new String[]{Manifest.permission.READ_EXTERNAL_STORAGE},
+                    1);
```

Comment:
Magic numbers! It's not clear that '1' and '2' in requestPermissions match '1' and '2' in onRequestpermissionsResult